tools/screenshots/thread-screenshot.js

```diff
@@ -64,17 +64,33 @@ async function run() {
         const messageListSelector = "#message-lists-container";
         await page.waitForSelector(messageListSelector);
 
+        // remove unread marker and don't select message
+        const marker = `.message-list[data-message-list-id="${CSS.escape(
+            message_list_id,
+        )}"] .unread_marker`;
+        await page.evaluate((sel) => {
+            $(sel).remove();
+        }, marker);
+
         const messageSelector = `#message-row-${message_list_id}-${CSS.escape(options.messageId)}`;
         await page.waitForSelector(messageSelector);
 
         const messageListBox = await page.$(messageListSelector);
         await page.evaluate((msg) => $(msg).removeClass("selected_message"), messageSelector);
 
+        // This is done so as to get white background while capturing screenshots.
+        const background_selectors = [".app-main", ".message-feed", ".message_header"];
+        await page.evaluate((selectors) => {
+            for (const selector of selectors) {
+                $(selector).css("background-color", "white");
+            }
+        }, background_selectors);
+
```

Comment:
This is to make sure there is no grey area when the messsagebox screenshot is captured (since it is curved, despite however close we try to capture the screenshot)